apps/judicial-system/backend/src/app/messages/rulingPdf.ts

```diff
@@ -24,11 +24,6 @@ export const ruling = {
     defaultMessage: 'Mættir eru:',
     description: 'Notaður sem fyrirsögn á lista yfir viðstadda.',
   }),
-  demandsHeading: defineMessage({
```

Comment:
Consider removing this in a separate PR, which would be merge in a later release.